Dockerfile

```diff
@@ -19,7 +19,9 @@ COPY . /work
 WORKDIR /work
 
 # Set up python env
-RUN pip install -r requirements-min.txt && pip install gunicorn
+RUN pip install "poetry==1.4.0"
+RUN poetry config virtualenvs.create false \
+  && poetry install --no-interaction --no-ansi
```

Comment:
what are the diff args here (no-interaction, no-ansi) doing?

Reply:
Fairly certain no-interaction is similar to -y, aka it won't look for user input.
no-ansi is supposed to be for easier reading of debug.